src/graphql/v1/schemas/custom/patient.graphql

```diff
@@ -1,11 +1,13 @@
 # import ExplanationOfBenefit from 'explanationOfBenefit.graphql'
 # import AllergyIntolerance from 'allergyIntolerance.graphql'
 # import CarePlan from 'carePlan.graphql'
+# import CareTeam from 'careTeam.graphql'
 
 extend type Patient {
     explanationOfBenefit: [ExplanationOfBenefit]
     allergyIntolerance: [AllergyIntolerance]
     carePlan: [CarePlan]
+    careTeam: [CareTeam]
```

Comment:
```suggestion
    careTeam(category: SearchToken): [CareTeam]
```
How are we handling filtering at this level? If we can't filter then we might as well use the root CareTeam query where we have filtering capabilities.